yarn.lock

```diff
@@ -14,20 +14,20 @@
     ts-poet "^4.9.0"
     typescript "^4.6.4"
 
-"@roaminroe/protoc@^2.4.0":
+"@roamin/protoc@^2.4.0":
   version "2.4.0"
-  resolved "https://registry.yarnpkg.com/@roaminroe/protoc/-/protoc-2.4.0.tgz#9f50b727af5bdf7a06e20ae38a2f9c80ef24a2d0"
-  integrity sha512-M2H/EaGYyNd4Zcskyf8PrtpMi7q7ewT+A7fPu3kIB3pN0y/8mXIuGRvqp8pO7LsKSCkGA+coCGTYeFLx7dC2lg==
+  resolved "https://registry.yarnpkg.com/@roamin/protoc/-/protoc-2.4.0.tgz#7d2d84f23aaac365024b3a14705f5605826e859a"
+  integrity sha512-vMVLvZ42wyQ3sd+vHzNAMGE2OSsURZ/CeuW3rnc8ueeNWhS1dVf9Nk6tQNd4GIINAMkjsBrl1eqdsmiaxV1epA==
 
 "@types/node@^17.0.34":
   version "17.0.45"
   resolved "https://registry.yarnpkg.com/@types/node/-/node-17.0.45.tgz#2c0fafd78705e7a18b7906b5201a522719dc5190"
   integrity sha512-w+tIMs3rq2afQdsPJlODhoUEKzFP1ayaoyl1CcnwtIlsVe7K7bA1NGm4s3PraqTLlXnbIN84zuBlxBWo1u9BLw==
 
-"as-proto@npm:@koinos/as-proto":
-  version "0.2.4"
-  resolved "https://registry.yarnpkg.com/@koinos/as-proto/-/as-proto-0.2.4.tgz#3ed5977578c35608e95f3459fed73b4bc416f513"
-  integrity sha512-vJnN9qQ+zbBOmPj5BREtaKwH2vS2aPFiuV4j4xPbUaaPWRU0mpPWyhYeTAsC5oh1Lf4oTNQ+y5LeUrzT/Y5aZQ==
+"as-proto@npm:@roamin/as-proto":
```

Comment:
Once koinos/as-proto#13 is merged, this can go back to pointing to `@koinos`, correct?

Reply:
The `yarn.lock` file is indeed invalid this is some leftover from my testing that I forgot to remove. Let's wait for https://github.com/koinos/as-proto/pull/13 to be merged and then I'll push the updated `yarn.lock`